lte/gateway/release/build-magma.sh

```diff
@@ -107,11 +107,8 @@ BUILD_DATE=`date -u +"%Y%m%d%H%M%S"`
 ARCH=amd64
 PKGFMT=deb
 MAGMA_PKGNAME=magma
-MAGMA_BUILD_PATH=${OUTPUT_DIR}/${MAGMA_PKGNAME}_${FULL_VERSION}_${ARCH}.${PKGFMT}
 SCTPD_PKGNAME=magma-sctpd
-SCTPD_BUILD_PATH=${OUTPUT_DIR}/${SCTPD_PKGNAME}_${FULL_VERSION}_${ARCH}.${PKGFMT}
 DHCP_CLI_PKGNAME=magma-dhcp-cli
-DHCP_CLI_BUILD_PATH=${OUTPUT_DIR}/${DHCP_CLI_PKGNAME}_${FULL_VERSION}_${ARCH}.${PKGFMT}
```

Comment:
Reason for the build errors: The path variables were moved up here in a refactoring but `OUTPUT_DIR` is not defined yet in this line.